content/en/docs/Responses/Limitation.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Limitation"
+linkTitle: "Limitation [OS]"
+opensubsonic:
+- Extension
+description: >
+  Defines a limitation for a codec profile.
+---
+
+{{< tabpane persist=false >}}
+{{< tab header="**Example**:" disabled=true />}}
+{{< tab header="OpenSubsonic" lang="json">}}
+{
+  "name": "audioBitrate",
+  "comparison": "LessThanEqual",
+  "value": "320000",
+  "required": true
+}
+{{< /tab >}}
+{{< /tabpane >}}
+
+| Field | Type | Req. | OpenS. | Details |
+| --- | --- | --- | --- | --- |
+| `name` | string | **Yes** | | The name of the limitation. Can be `audioChannels`, `audioBitrate`, `audioProfile`, `audioSamplerate`, or `audioBitdepth`. |
+| `comparison` | string | **Yes** | | The comparison operator. Can be `Equals`, `NotEquals`, `LessThanEqual`, `GreaterThanEqual`, `EqualsAny`, or `NotEqualsAny`. |
+| `value` | string | **Yes** | | The value to compare against. For `EqualsAny` and `NotEqualsAny`, this should be a pipe-separated (`\|`) list of values (e.g.,  `44100\|48000`). |
+| `required` | boolean | **Yes** | | Whether this limitation must be met. |
```

Comment:
What would be the point of a limitation that is not required?

Reply:
For optimizations clients side and directplay vs transcoding limitations.

Like the app can play 32b flac direct, but if you transcode then I'd like 16b flac.

Reply:
Maybe I missed something: how do you specify this since a codec profile is not attached to neither a direct play profile nor a transcoding profile?